src/main/java/net/minecraftforge/common/ForgeMod.java

```diff
@@ -389,10 +391,9 @@ public ForgeMod(FMLJavaModLoadingContext context) {
         });
         modEventBus.addListener(this::preInit);
         modEventBus.addListener(this::gatherData);
-        modEventBus.addListener(this::loadComplete);
+//        modEventBus.addListener(this::loadComplete);
```

Comment:
If you do not plan on using `loadComplete(FMLLoadCompleteEvent)` upon finishing this PR, please remove this one.